packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -101,13 +101,14 @@ export interface OutputBuilder {
    * @returns {Promise<MaybeValidTxOut>} When it is a `ValidTxOut` it can be used as input in `TxBuilder.addOutput()`.
    * In case of it is an `InvalidTxOut`, it embeds a TxOutValidationError with more details
    */
+  // REVIEW: let's remove MaybeValidTxOut and rely on promise throwing
```

Comment:
Please review this

Reply:
Agree, error handling should be consistent with TxBuilder

Reply:
[87f8da3](https://github.com/input-output-hk/cardano-js-sdk/pull/712/commits/87f8da333ebc7f2c94a4b7d04f7ef7f51dab619b)